wcomponents-app-archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -30,37 +31,68 @@
 				</configuration>
 			</plugin>
 		</plugins>
-	</build>   
+	</build>
 
-	<dependencies>
-  
-		<dependency>
-			<groupId>com.github.bordertech.wcomponents</groupId>
-			<artifactId>wcomponents-core</artifactId>
-			<version>${bordertech.wcomponents.version}</version>
-		</dependency>
+    <dependencies>
 
-		<dependency>
-			<scope>test</scope>
-			<groupId>com.github.bordertech.wcomponents</groupId>
-			<artifactId>wcomponents-test-lib</artifactId>
-			<version>${bordertech.wcomponents.version}</version>
-		</dependency>
+	<dependency>
+		<groupId>com.github.bordertech.wcomponents</groupId>
+		<artifactId>wcomponents-core</artifactId>

```

Comment:
Fixes look good.

You could use the new "wcomponents-bundle" module, that brings in both "core" and the "theme".